src/services/BoardApi.js

```diff
@@ -163,30 +163,32 @@ export class BoardApi {
 					let CSV = row + '\r\n'
 
 					response.data.stacks.forEach(stack => {
-						stack.cards.forEach(card => {
-							row = ''
-							Object.keys(fields).forEach(field => {
-								if (field === 'createdAt' || field === 'lastModified') {
-									const date = new Date(Number(card[field]) * 1000)
-									row += '"' + date.toLocaleDateString() + '"' + '\t'
-								} else if (field === 'stackId') {
-									row += '"' + stack.title + '"' + '\t'
-								} else if (field === 'labels') {
-									row += '"'
-									card[field].forEach(label => {
-										row += label.title + ', '
-									})
-									if (card[field].length > 0) {
-										row = row.slice(0, -1)
+						if (stack.cards) {
+							stack.cards.forEach(card => {
```

Comment:
Thanks a lot for your contribution.

I would suggest to avoid the extra if branch. We can use optional chaining to not call forEach if cards is not set:

```js
stack?.cards?.forEach(card => {
```

That way the change is much smaller and less nesting means less complexity when reading the code.

Reply:
Sure, I've made the change - I was not aware of this syntax.